packages/flutter_test/lib/src/controller.dart

```diff
@@ -139,10 +169,10 @@ class SemanticsController {
   ///   parts of the traversal.
   /// * [orderedEquals], which can be given an [Iterable<Matcher>] to exactly
   ///   match the order of the traversal.
-  Iterable<SemanticsNode> simulatedAccessibilityTraversal({Finder? start, Finder? end}) {
+  Iterable<SemanticsNode> simulatedAccessibilityTraversal({Finder? start, Finder? end, FlutterView? view}) {
```

Comment:
I would have thought that the view is implied by start/end. Whatever view these are in, that's the view so use. Why do we need the option to specify a separate view?

Reply:
If we do need this: the doc comment above should mention it.

Reply:
Neither `start` nor `end` are required. The call `simulatedAccessibilityTraversal();` currently returns a traversal of the entire semantics tree. This allows for the same behavior, but supports multi-view where they _would_ have to specify a view to get the right semantics tree (eventually).

Reply:
I did add in an explanation of the `view` parameter as requested.

Reply:
Removed the new parameter and new documentation as discussed in GVC.